unmarshal.go

```diff
@@ -33,15 +33,14 @@ var (
 )
 
 // Unmarshal EBML stream
-func Unmarshal(r io.Reader, val interface{}) error {
+func Unmarshal(r io.Reader, val interface{}, opts ...UnmarshalOptions) error {
```

Comment:
```suggestion
func Unmarshal(r io.Reader, val interface{}, opts ...UnmarshalOption) error {
	options := &UnmarshalOptions{}
	for _, o := range opts {
		o(options)
	}
```
Then, `options` can be used instead of `opts...`.

Reply:
Ah, sorry. I misunderstood about that. 
Fixed by 3db47e12fd5cd5b7894500c071a54a7be7b7c156